.github/workflows/continuous-build.yml

```diff
@@ -0,0 +1,61 @@
+# MIT License
+#
+# Copyright (c) 2019 Choko (choko@curioswitch.org)
```

Comment:
Is this file copied from Curiostack? If so, the current copyright statement is right. 
If you wrote this file by yourself, the copyright should be Infostellar.

Reply:
It's copied from Curiostack in its entirety.

Reply:
That's fine in that case. Thanks.